src/common/Navigation/index.js

```diff
@@ -21,7 +21,10 @@ const Navigation = () => {
           <StyledLogo>
             <StyledIcon />
             <Title>
-              <StyledLink>
+              <StyledLink
+                href="/"
+                rel="noreferrer"
```

Comment:
teraz poczytałem o tym `noreferrer`, używa się go raczej do linkowania zewnętrznych stron, żeby nie przesyłać informacji. Jak linkujemy do naszej to nie trzeba tego pisać 😁